common/src/main/java/org/apache/nemo/common/ir/IRDAG.java

```diff
@@ -110,8 +109,16 @@ public boolean advanceDAGSnapshot(final BiFunction<IRDAG, IRDAG, Boolean> checke
     return canAdvance;
   }
 
+  /**
+   * @return a IR DAG summary string, consisting of only the original vertices generated from the frontend.
```

Comment:
What is the original vertex? could you please define what is original or not?

Reply:
There's been a slight miswording. I've changed the comments. Thanks!